static/css/_landing-binary.scss

```diff
@@ -840,4 +730,101 @@ $header-color : #333333;
 
 }
 
+.background-x {
```

Comment:
I'm not familiar with SASS much and not sure about the naming convention in bot project, Would someone let me know how should I name my styles please 🙏